### DISCUSSION TITLE
Custom interface's options default values aren't set as v-form values?

### DISCUSSION THREAD

**Discussion by brutalv4** [directus/directus#5322] [Archive]
Hi!
I'm working on the custom Map interface for the v9 (port from v8).
It seems that default schema values from options aren't set as form values and afterwards will not be saved into the directus_fields table.
Am I missing something?

![Screen-2021-04-28_14-20-06](https://user-images.githubusercontent.com/6163711/116396305-e42bc300-a82d-11eb-9e23-dc6756b9424f.jpg)

Thank you.

**Comment by rijkvanzanten** [comment]
This is correct. This is because of the fact that options in that index context can either be an array of fields, or a custom Vue component. Right now, the interface itself can define it's default values in the props it listens to 👍🏻

**Reply by rijkvanzanten** [reply]
(cc @nickrum)
